packages/select/src/Select.spec.tsx

```diff
@@ -811,5 +813,32 @@ describe('packages/select', () => {
         expect(getByTextFor(combobox, 'selected')).toBeVisible();
       });
     });
+
+    describe('when the "state" is "error"', () => {
+      test('displays error message when input is invalid', () => {
```

Comment:
This is trivial, but I think this test name should mention the `state` prop, and not the input (since it technically has nothing to do with the selected value)